sdk/ai/azure-ai-client/azure/ai/client/models/_patch.py

```diff
@@ -259,7 +258,7 @@ def execute(self, tool_call: RequiredFunctionToolCall) -> Any:
             raise
 
     @property
-    def definitions(self) -> Sequence[FunctionToolDefinition]:
+    def definitions(self) -> List[ToolDefinition]:
```

Comment:
Does Pylance look good with this change?

Reply:
Yes.    I also alerted the error before you changed from List to Sequence took 10 mins to show up.   After making this change, I ensure I waited long enough, and it doesn't say "Loading".   And still no error.